tools/spark.md

```diff
@@ -12,6 +12,10 @@ and, when combined with Spark, enable richer analysis than SQL alone.
 The Spark clusters can be launched from [ATMO](https://analysis.telemetry.mozilla.org).
 The Spark Python API is called PySpark.
 
+Note that this documentation focuses on ATMO, but analysis with Spark is also possible using DataBricks.
```

Comment:
Looks like we're standardizing around "Databricks" instead of "DataBricks"

https://github.com/mozilla/firefox-data-docs/blob/master/.spelling#L38